Client/Frontend/Widgets/AuralProgressBar.swift

```diff
@@ -68,18 +69,40 @@ class AuralProgressBar {
             engine.disconnectNodeOutput(progressPlayer)
         }
 
-        @objc func audioEngineConfigurationDidChange(notification: NSNotification) {
-            disconnectPlayerNodes()
-            connectPlayerNodes()
-
+        func startEngine() {
             if !engine.running {
-                do { try engine.start() }
-                catch {
-                    log.error("Unable to restart AVAudioEngine. Player will not automatically restart : \(error)")
+                do {
+                    try engine.start()
+                } catch {
+                    log.error("AuralPlayer.startEngine: Unable to start AVAudioEngine: \(error)")
```

Comment:
This doesn't need the log tag decoration: `log` will log the function name and line number for you.